platform-hub-web/src/app/projects/projects-list.component.js

```diff
@@ -23,18 +26,66 @@ function ProjectsListController(roleCheckerService, Projects) {
   function loadProjects() {
     ctrl.loading = true;
 
-    Projects
-      .refresh()
-      .finally(() => {
-        ctrl.loading = false;
+    return Projects.getAll()
+    .then(projects => {
+      return Me.refresh().then(() => {
+        const currentUserId = Me.data.id;
+        return Promise.all(projects.map(projectIdto => {
+          return Projects.getMemberships(projectIdto.id);
+        })).then(memberships => {
+          ctrl.memberships = memberships;
+
+          ctrl.myProjects = getMyProjects(projects, currentUserId, memberships);
+          ctrl.notMyProjects = getNotMyProjects(projects, currentUserId, memberships);
+
+          ctrl.myProjects = _.orderBy(ctrl.myProjects, [project => project.name.toLowerCase()], ['asc']);
```

Comment:
may as well move this inside `getProjects` if refactoring to single function as mentioned in above comment^

Reply:
Resolved - moved into refactored getProjects function